CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-# Unreleased
+# 68.0.1
```

Comment:
This should probably be a minor bump as this add functionality, rather than just a bug fix.
```suggestion
# 68.1.0
```

Reply:
Sorry, fixed